src/utils/test/mocks.js

```diff
@@ -142,7 +142,7 @@ const createExtensionInput = () => ({
   appDir: '/path/to/project-name',
   assetsPath: '/path/to/project-name/assets',
   workflowsFolder: '/path/to/project-name/workflows',
-  features: features,
+  features: getFeatures(true),
```

Comment:
```suggestion
  features: getFeatures(),
```
because of:
`isPip3Avaialble = true` in `module.exports = (isPip3Avaialble = true) => [`